### PR TITLE
SISRP-16500 - Retrofit CARS Activity Front-end for transition, incorporating feature flag

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-16500

### CODE REVIEW

File: src/assets/stylesheets/_myfinances.scss [hunk 1]
```diff
@@ -20,6 +20,9 @@
       margin-right: 10px;
     }
   }
+  .cc-page-myfinances-account-summary-alert {
+    margin: 14px 20px 0 15px;
```
Comment (christianvuerings) [inline src/assets/stylesheets/_myfinances.scss]: @davidlimcheng could we make this `margin: 15px 20px 0 15px`? Or is there a specific reason for the `14px`?

File: src/assets/templates/cars_summary.html [hunk 2]
```diff
@@ -1,12 +1,19 @@
 <div class="cc-widget" data-ng-controller="CarsController">
   <div class="cc-widget-title">
-    <h2 class="cc-left">Billing Summary</h2>
+    <h2 class="cc-left">
+      Billing Summary
+      <span data-ng-if="api.user.profile.features.csBilling" class="cc-widget-title-sub">PRIOR TO FALL 2016</span>
```
Comment (christianvuerings) [inline src/assets/templates/cars_summary.html]: @davidlimcheng Could we use the class `cc-text-uppercase` on the uppercase text and use normal case in the html?

**Comment by christianvuerings** [comment]
Lgtm as soon as comments are addressed.

File: src/assets/stylesheets/_myfinances.scss (hunk 1 above)
Comment (davidlimcheng) [inline src/assets/stylesheets/_myfinances.scss]: @christianvuerings No reason, I set it to `8px` (same as `cc-page-myfinances-account-summary-item`) and just eyeballed it from there to make it similar to the design.  Will set it to 15px.

File: src/assets/templates/cars_summary.html (hunk 2 above)
Comment (davidlimcheng) [inline src/assets/templates/cars_summary.html]: @christianvuerings Sure thing, and I'll be sure to follow suit on future uppercased texts.

**Comment by davidlimcheng** [comment]
Changes have been made, thanks @christianvuerings !

**Pull request merged**